packages/now-cli/src/commands/inspect.js

```diff
@@ -214,95 +178,7 @@ export default async function main(ctx) {
     print(`\n\n`);
   }
 
-  if (limits) {
```

Comment:
I think we still have limits for v2, although I'm not sure if these are included.

See the [`functions`](https://vercel.com/docs/configuration#project/functions) property for an example.

Reply:
Confirmed on Slack that these are V1 limits (Cloud V2)

Reply:
Nate confirmed, these limits are v1 only